pkg/features/kube_features.go

```diff
@@ -167,6 +167,12 @@ const (
 	//
 	// Enable pods to consume pre-allocated huge pages of varying page sizes
 	HugePages utilfeature.Feature = "HugePages"
+    
+        // owner: @lmdaly
+	// alpha: v1.13
```

Comment:
If we are just adding this now, this should be 1.14/1.15, right?

Reply:
Changed to 1.15